src/boot/measure.c

```diff
@@ -726,7 +726,6 @@ static int verb_sign(int argc, char *argv[], void *userdata) {
         _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
```

Comment:
a rationale for this patch would be good in the commit msg

Reply:
will update in next push, thanks